docker-compose.yaml

```
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+  backend:
+    build: .
+    container_name: healoscope-backend
+    ports:
+      - "8080:8080"
+    environment:
+      - PORT=8080
+      - MONGO_URI=mongodb://mongo:27017/healoscope
```

also include the other env vars required that you've mentioned in README. you can simply do `GEMINI_API_KEY: ${GEMINI_API_KEY}` and specify values in `.env` file and it'll be automatically read here. Let port and MONGODB_URI alone be hardcoded here as they are docker hostname specific.
  

```suggestion
      PORT: 8080
      MONGODB_URI: mongodb://mongo:27017/healoscope
```